src/Pages/Community/CommunityDetail/CommunityDetailMain/CommunityDetailMain.scss

```
@@ -0,0 +1,23 @@
+@import "../../../../Styles/common.scss";
+
+main.CommunityDetailMain {
+  width: 830px;
+  border-right: 1px solid #ededed;
+
+  .mainHeader {
+    @extend %flex-align-center;
+    height: 90px;
+
+    span {
+      padding-right: 5px;
+      padding-left: 5px;
```

```suggestion
      padding: 0 5px;
```
속성을 한 줄로 줄여줄 수 있겠네요!